lib/ui/widgets/components/app_button_tiny.dart

```diff
@@ -12,6 +13,57 @@ import 'package:flutter_vibrate/flutter_vibrate.dart';
 
 enum AppButtonTinyType { primary, primaryOutline }
 
+class AppButtonTinyConnectivity extends ConsumerWidget {
+  const AppButtonTinyConnectivity(
+    this.buttonText,
```

Comment:
Maybe you should add a `AppButtonTinyType type` property. That property would be simply passed to `AppButtonTiny` constructor.